R/Spectra.R

```diff
@@ -989,6 +987,15 @@ setReplaceMethod("dataOrigin", "Spectra", function(object, value) {
 #' @rdname Spectra
 setMethod("dataStorage", "Spectra", function(object) dataStorage(object@backend))
 
+#' @rdname Spectra
+setMethod("dropNaSpectraVariables", "Spectra", function(object) {
```

Comment:
This method is nearly identical to `dropNaSpectraVariables,MsBackend`. Wouldn't be the following sufficient:
```r
setMethod("dropNaSpectraVariables", "Spectra", function(object) {
    object@backend <- dropNaSpectraVariables(object@backend)
    object
}
```

Another solution would be to drop  `dropNaSpectraVariables,MsBackend` (and keep  `dropNaSpectraVariables,Spectra`) because it isn't used anywhere, or?

Reply:
thanks for the suggestion. The `dropNaSpectraVariables,MsBackend` isn't used anywhere yet, but I find it quite usefull and we might use/call it also in the backend at some point.

Reply:
Done.